docs/API_SPECS_SIMULATIONS.md

```diff
@@ -617,4 +617,4 @@ Simulations can be deleted using the DELETE method.
 
 ```
 DELETE /v1/simulations/1
-```
\ No newline at end of file
+```
```

Comment:
same as above: what is the change here? if nothing has changed then revert to master version for this file to keep history clean

Reply:
done